react-validated-input/my-app/src/ValidatedInput.tsx

```diff
@@ -0,0 +1,46 @@
+import { useState } from 'react';
+import { FaCheck } from 'react-icons/fa';
+import { CgClose } from 'react-icons/cg';
+import './ValidatedInput.css';
+
+export function ValidatedInput() {
+  // char is a string of characters like an array
+  const [char, setChar] = useState('');
+  let textContent = '';
+  let Xcheck = '';
+  let myCheck = '';
+
+  if (char === '') {
+    textContent = 'A password is required';
+    Xcheck = 'xcheck';
+    myCheck = 'hidden';
+  } else if (Number(char.length) < 8) {
+    textContent = 'password is too short';
+    Xcheck = 'xcheck';
+    myCheck = 'hidden';
+  } else if (Number(char.length) === 8) {
+    Xcheck = 'hidden';
+    myCheck = 'my-check';
+  }
```

Comment:
Using the `hidden` CSS attribute works for this, but an easier way is to use conditional rendering. That is, you only render the icon into the DOM if it should be drawn. A ternary in the JSX is a good way to do that.